src/groq/index.ts

```diff
@@ -111,4 +124,38 @@ export default class GroqChat {
       }
     }
   }
+
+  /**
+   * Logs the token usage information to the console. by args --token-usage or -tu
+   *
+   * @param tokenUsageInfo - The token usage information object.
+   */
+  public accumulateToken(tokenUsageInfo: Groq.CompletionUsage): void {
```

Comment:
suggestion: this function should be `private` because it's only being used in this class -- it should also be moved up because standard says:
- Private functions first
- Public functions second